weaviate/classes/query.py

```diff
@@ -25,6 +26,7 @@
     "HybridFusion",
     "MetadataQuery",
     "Move",
+    "PhoneNumberType",
```

Comment:
do we need to export it here? It cannot be used for querying?

Reply:
Ah, you're right, this should be exported in a `weaviate/types/*.py` file. Will change and then we can use that as the place to export other types as mentioned above